docker/demo.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 MNEMONIC="volcano story trust file before member board recycle always draw fiction when"
```

Comment:
It is needed to provide an explication about the values of this MNEMONIC

Reply:
I suggest to explain it in the README file

Reply:
I know but there was no details about from where that information was generated and it is important to know it.